README.md

```diff
@@ -1,9 +1,9 @@
-# The Algorithms - Java
+# The Algorithms of - Java
```

Comment:
**The algorithms** is the project name and **Java** is the name of repository in it. The orignal `# The Algorithms - Java` was correct. Correct this change.